packages/react/src/MegaMenu/MegaMenu.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren } from 'react'
+
+export interface MegaMenuProps extends PropsWithChildren<HTMLAttributes<HTMLElement>> {}
+
+export const MegaMenu = forwardRef(
+  ({ children, className, ...restProps }: MegaMenuProps, ref: ForwardedRef<HTMLElement>) => (
+    <nav {...restProps} ref={ref} className={clsx('amsterdam-mega-menu', className)}>
```

Comment:
I'd keep this a div for now, if MegaMenu is going to be a child of PageMenu, the entire thing should be wrapped in a `nav`

Reply:
Probably a sibling of header but i'll make it a div.

Reply:
I think the general pattern is to wrap everything (PageMenu + MegaMenu) in a `nav`, see also [Accessible Mega Menu](https://adobe-accessibility.github.io/Accessible-Mega-Menu/)